lang/pl/index.md

```diff
@@ -1,42 +1,43 @@
 ---
-title: Semantyczne Wersjonowanie 2.0.0
+title: Wersjonowanie semantyczne 2.0.0
 language: pl
 ---
 
-Semantyczne Wersjonowanie 2.0.0
-==============================
+Wersjonowanie semantyczne 2.0.0
+===============================
 
 Streszczenie
--------
+------------
 
-Mając do dyspozycji konkretny numer MAJOR.MINOR.PATCH, należy dokonać inkrementacji:
+Dla numeru wersji MAJOR.MINOR.PATCH, zwiększaj:
 
-1. MAJOR wersji, gdy dokonujesz zmian niekompatybilnych z API,
-1. MINOR wersji, gdy dodajesz nową funkcjonalność, która jest kompatybilna z poprzednimi wersjami,
-1. PATCH wersji, gdy dokonujesz naprawy błędu, która jest kompatybilna z poprzednimi wersjami.
+1. wersję MAJOR, gdy dokonujesz zmian niekompatybilnych z API,
+1. wersję MINOR, gdy dodajesz nową funkcjonalność, która jest kompatybilna z poprzednimi wersjami,
+1. wersję PATCH, gdy naprawiasz błąd w sposób kompatybilny z poprzednimi wersjami.
```

Comment:
It should be `gdy naprawiasz błąd nie zrywając kompatybilności z poprzednimi wersjami.`